scripts/ci/build-and-test.sh

```diff
@@ -1,57 +1,28 @@
 #!/usr/bin/env bash
-set -ex
+#set -ex
```

Comment:
It adds a lot of noise to the logs since we already are logging a lot of stuff. This file is relatively small.